swagger/swagger-invocation/invocation-validator/src/main/java/org/apache/servicecomb/swagger/invocation/validator/ParameterValidator.java

```diff
@@ -16,8 +16,10 @@
  */
 package org.apache.servicecomb.swagger.invocation.validator;
 
+import java.util.Map;
 import java.util.Set;
 
+import org.apache.servicecomb.config.ConfigUtil;
```

Comment:
delete used ParameterValidator and ProducerInvokeExtension

Reply:
delete